examples/example_a1_walking/main.cpp

```diff
@@ -0,0 +1,172 @@
+/* ----------------------------------------------------------------------------
+ * GTDynamics Copyright 2020, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * See LICENSE for the license information
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file  main.cpp
+ * @brief Spider trajectory optimization with pre-specified footholds.
+ * @author: Dan Barladeanu
+ */
+
+#include <gtdynamics/factors/ObjectiveFactors.h>
+#include <gtdynamics/universal_robot/sdf.h>
+#include <gtdynamics/utils/Trajectory.h>
+#include <gtsam/nonlinear/LevenbergMarquardtOptimizer.h>
+#include <gtsam/nonlinear/NonlinearFactorGraph.h>
+#include <gtsam/base/timing.h>
+
+#include <algorithm>
+#include <boost/algorithm/string/join.hpp>
+#include <boost/optional.hpp>
+#include <fstream>
+#include <iostream>
+#include <utility>
+
+using std::string;
+using std::vector;
+
+using gtsam::Point3;
+using gtsam::Pose3;
+using gtsam::Rot3;
+using gtsam::Vector6;
+using gtsam::noiseModel::Isotropic;
+using gtsam::noiseModel::Unit;
+
+using namespace gtdynamics;
+
+// Returns a Trajectory object for a single spider walk cycle.
```

Comment:
Spider? ;-)

Reply:
I had a little trouble with the a1 sdf which was in the repo already. It wouldn't load in pybullet at all.